recipe-server/client/control_new/components/recipes/RecipeListing.js

```diff
@@ -156,9 +158,27 @@ export default class RecipeListing extends React.Component {
     push(getCurrentURL({ page }));
   }
 
-  handleRowClick(record) {
-    const { push } = this.props;
-    push(`/recipe/${record.id}`);
+  handleRowClick(record, index, event) {
+    const linkTarget = event.target.tagName === 'A';
+
+    // If the user has clicked a link directly, just fall back to the native event.
+    if (linkTarget) {
+      return;
+    }
+
+    // If we're here, the user clicked the row itself, which now needs to behave
+    // as if it was a native link click. This includes opening a new tab if using
+    // a modifier key (like ctrl).
+
+    const usingModifierKey = (event.ctrlKey || event.metaKey || event.button === 1);
+    let navTo = this.props.push;
+
+    // No link but using a modifier key = open in a new tab.
+    if (!linkTarget && usingModifierKey) {
```

Comment:
`linkTarget` can't ever be true here, since if it was you would have bailed early in the condition above.